### PR TITLE
add hami to Kubernetes device plugin examples

### PR DESCRIPTION
**Pull request by wawa0210**
### Description

add hami to Kubernetes device plugin examples

### Issue

Closes: #

### CODE REVIEW

**Comment by T-Lakshmi** [comment]
Hello @wawa0210,
Thanks for your contribution. It appears that this PR includes changes to files across multiple languages English (en) and Chinese (zh-cn). However, docs follows different processes for each localization, and we typically don't accept pull requests that impact multiple languages simultaneously. I would recommend modify only in primary language(English) initially with this PR. After the merge of the English content, you can create separate pull request for the remaining language(s).

**Comment by T-Lakshmi** [comment]
Explicit hold until this PR only covers one language.
(Ok to unhold once that's done)
/hold

**Comment by wawa0210** [comment]
> Explicit hold until this PR only covers one language. (Ok to unhold once that's done) /hold

Thanks for the reminder, I have changed to a one language

File: content/en/docs/concepts/extend-kubernetes/compute-storage-net/device-plugins.md [hunk 1]
```diff
@@ -486,6 +486,7 @@ Here are some examples of device plugin implementations:
 * [Akri](https://github.com/project-akri/akri), which lets you easily expose heterogeneous leaf devices (such as IP cameras and USB devices).
 * The [AMD GPU device plugin](https://github.com/ROCm/k8s-device-plugin)
 * The [generic device plugin](https://github.com/squat/generic-device-plugin) for generic Linux devices and USB devices
+* [HAMi](https://github.com/Project-HAMi/HAMi),Heterogeneous AI Computing Virtualization Middleware,Support unified scheduling, management, and monitoring of heterogeneous AI chips(such as Nvidia,Cambricon,Hygon,Iluvatar,Mthreads,Ascend,Metax)
```
Comment (windsonsea) [inline content/en/docs/concepts/extend-kubernetes/compute-storage-net/device-plugins.md:489]:
```suggestion
* [HAMi](https://github.com/Project-HAMi/HAMi), formerly known as 'k8s-vGPU-scheduler', is a
  heterogeneous device management middleware for Kubernetes. It supports diverse AI chips,
  including Nvidia, Cambricon, Hygon, Iluvatar, Mthreads, Ascend, and Metax.
```

**Comment by wawa0210** [comment]
Thanks your review, already update.

File: content/en/docs/concepts/extend-kubernetes/compute-storage-net/device-plugins.md (hunk 1 above)
Comment (windsonsea) [reply content/en/docs/concepts/extend-kubernetes/compute-storage-net/device-plugins.md]: Do not mark it as 'Resolved' if any differing opinions. We value and encourage diverse ideas! 😅

Comment (wawa0210) [reply content/en/docs/concepts/extend-kubernetes/compute-storage-net/device-plugins.md]: It should be a mistake, not intentional. I have simplified the description, can you help review it again? Thank you.

File: content/en/docs/concepts/extend-kubernetes/compute-storage-net/device-plugins.md [hunk 2]
```diff
@@ -486,6 +486,7 @@ Here are some examples of device plugin implementations:
 * [Akri](https://github.com/project-akri/akri), which lets you easily expose heterogeneous leaf devices (such as IP cameras and USB devices).
 * The [AMD GPU device plugin](https://github.com/ROCm/k8s-device-plugin)
 * The [generic device plugin](https://github.com/squat/generic-device-plugin) for generic Linux devices and USB devices
+* The [HAMi](https://github.com/Project-HAMi/HAMi) for heterogeneous ai computing virtualization middleware. (For example, NVIDIA, Cambricon, Hygon, Iluvatar, MThreads, Ascend, Metax.)
```
Comment (katcosgrove) [inline content/en/docs/concepts/extend-kubernetes/compute-storage-net/device-plugins.md:489]: Grammatical correction. After that, you're good to go

```suggestion
* The [HAMi](https://github.com/Project-HAMi/HAMi) for heterogeneous AI computing virtualization middleware (for example, NVIDIA, Cambricon, Hygon, Iluvatar, MThreads, Ascend, Metax)
```

Comment (wawa0210) [reply content/en/docs/concepts/extend-kubernetes/compute-storage-net/device-plugins.md]: thanks your review, done

**Review by sftim — COMMENTED** [review]
Thanks

/lgtm
/approve

**Pull request merged**